src/app/shared/add-entity-popup/add-entity-popup.component.ts

```
@@ -0,0 +1,110 @@
+  let url = urlConfig.observation.searchEntities + this.observationId + `&search=${searchValue}`;
+
+  if (parentEntityId) {
+    url += `&parentEntityId=${parentEntityId}`;
+  }
+
+  this.apiService.post(url, this.apiService.profileData).pipe(finalize(() => this.loaded = false))
+      .subscribe((res: any) => {
+        if (res.result) {
+          const searchEntities = res?.result[0];
+          this.searchEntities = searchEntities?.data;
+          this.filteredEntities = [...searchEntities?.data]
```

Biome (2.1.2)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)